google/resource_container_cluster.go

```diff
@@ -409,6 +424,12 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.Description = v.(string)
 	}
 
+	if _, ok := d.GetOk("legacy_abac"); ok {
```

Comment:
In addition, `LegacyAbac.Enabled` is a boolean that's marshaled with `omitempty`, so even if we do go in here, we won't send `false` unless this field is added to the `LegacyAbac.ForceSendFields` slice